### PR TITLE
Appstest

### PR DESCRIPTION
**Pull request by bharathsreenivas**

### CODE REVIEW

**Comment by bharathsreenivas** [comment]
Adding scripts

File: hdinsight-linux-add-hue-on-existing-cluster/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "clusterName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the existing HDInsight cluster"
+            }
+        },
+        "galleryPackageIdentity": {
```
Comment (travismc1) [inline hdinsight-linux-add-hue-on-existing-cluster/azuredeploy.json]: Let's hardcode galleryPackageIdentity and make as a variable

File: hdinsight-linux-with-hue-on-edge-node/azuredeploy.json [hunk 2]
```diff
@@ -1,88 +1,95 @@
 {
```
Comment (travismc1) [inline hdinsight-linux-with-hue-on-edge-node/azuredeploy.json:4]: Let's rename to hdinsight-linux-with-hue-application?

File: hdinsight-linux-with-hue-on-edge-node/azuredeploy.json [hunk 3]
```diff
@@ -1,88 +1,95 @@
...
+      }
+    },
+    "clusterWorkerNodeCount": {
+      "type": "int",
+      "defaultValue": 2,
+      "metadata": {
+        "description": "The number of nodes in the HDInsight cluster."
+      }
+    },
+   "galleryPackageIdentity": {
```
Comment (travismc1) [inline hdinsight-linux-with-hue-on-edge-node/azuredeploy.json]: same comment as the other template

File: hdinsight-linux-add-hue-on-existing-cluster/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+                    "name": "edgenode",
+                    "targetInstanceCount": 1,
+                    "hardwareProfile": {
+                        "vmSize": "[parameters('edgeNodeSize')]"
+                    }
+                }]
+            },
+            "installScriptActions": [{
+                "name": "[concat('hue-install-v0','-' ,uniquestring(variables('applicationName')))]",
+                "uri": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/Hue/scripts/Hue-install_v0.sh",
```
Comment (travismc1) [inline hdinsight-linux-add-hue-on-existing-cluster/azuredeploy.json]: Validate this path; The hue script should be point to this same file, /hdinsight-linux-hue-with-edgenode/scripts/Hue-installv0.sh - should point to one in PR

File: hdinsight-linux-with-hue-on-edge-node/azuredeploy.json [hunk 5]
```diff
@@ -91,47 +98,134 @@
...
+                "roles": [{
+                    "name": "edgenode",
+                    "targetInstanceCount": 1,
+                    "hardwareProfile": {
+                        "vmSize": "[parameters('edgeNodeSize')]"
+                    }
+                }]
+            },
+            "installScriptActions": [{
+                "name": "[concat('hue-install-v0','-' ,uniquestring(variables('applicationName')))]",
```
Comment (travismc1) [inline hdinsight-linux-with-hue-on-edge-node/azuredeploy.json]: same thing about relative path

File: hdinsight-linux-with-hue-on-edge-node/metadata.json [hunk 6]
```diff
@@ -1,7 +1,7 @@
 {
   "itemDisplayName": "Create HDInsight Linux Cluster with Hue app on Edge Node",
   "description": "Template creates an HDInsight Linux cluster in a virtual network with another vm as an edge node that is bootstrapped with the clusters configurations and Hue is installed on edge node.",
```
Comment (travismc1) [inline hdinsight-linux-with-hue-on-edge-node/metadata.json]: Let's change the description and summary. Template creates HDI cluster and installs Hue on an edgenode.

**Comment by travismc1** [comment]
The new readme file is mostly good; let's just add links to our public docs. 
https://azure.microsoft.com/en-us/documentation/articles/hdinsight-apps-install-custom-applications/
Copy the new readme file (text not only, not the links) to the older one.
Also, you'll need to change the deploy links.

File: hdinsight-linux-add-hue-on-existing-cluster/README.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
+# Install Hue as an HDInsight Iaas Cluster Application
+
+Installs Hue as an Iaas Cluster Application on an existing cluster -<br>
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fhdinsight%2FIaas-Applications%2Fmaster%2FHue%2Fazuredeploy.json" target="_blank">
```
Comment (travismc1) [inline hdinsight-linux-add-hue-on-existing-cluster/README.md]: /uri/UriToTemplate with escaped. Temporary test and change stuff with azure.

File: hdinsight-linux-with-hue-application/azuredeploy.parameters.json [hunk 8]
```diff
@@ -5,26 +5,29 @@
...
+        "clusterWorkerNodeCount": {
             "value": "changeme"
         },
         "sshUserName": {
             "value": "changeme"
         },
         "sshPassword": {
             "value": "changeme"
+        },
+        "edgeNodeSize": {
```
Comment (travismc1) [inline hdinsight-linux-with-hue-application/azuredeploy.parameters.json:29]: Check that the parameters file match the parameters in both locations

**Comment by travismc1** [comment]
Also copy the Hue install script so that a copy is in each folder and change the links.
After that change all the links to the azurequickstart repo and submit a PR to them

**Pull request merged**